src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,460 @@
+            fromYear = fromYear.substring(0, fromYear.indexOf(" "));
+            LocalDate from = LocalDate.of(Integer.parseInt(fromYear), Integer.parseInt(fromMonth), Integer.parseInt(fromDate));
+            Event event = new Event(content, from, secondTo);
+            //System.out.println(event);
+            ui.add(event, list.size(), onetwo);
+            list.add(event);
+            //System.out.println("Now you have " + list.size() + onetwo +  " in the list");
+            //System.out.println("____________________________________________________________");
+            appendToFile(zenithData, parser.getStr());
+            refreshData();
+            Answer();
+
+        } else if (parser.getArr()[0].equals("deadline")) {
+            arr = parser.getArr();
+            String by = arr[1].substring(arr[1].indexOf("/by") + 4);
+            //System.out.println("____________________________________________________________");
+            //System.out.println("Got it. I've added this task:");
+            String byDate = by.substring(0, by.indexOf("/"));
+            String firstByMonth = by.substring(by.indexOf("/") + 1);
+            String byMonth = firstByMonth.substring(0, firstByMonth.indexOf("/"));
+            String byYear = firstByMonth.substring(firstByMonth.indexOf("/") + 1);
+
+
+            LocalDate secondBy = LocalDate.of(Integer.parseInt(byYear), Integer.parseInt(byMonth), Integer.parseInt(byDate));
+
+            Deadline deadline = new Deadline(arr[1].substring(0, arr[1].indexOf("/by ")), secondBy);
+            //System.out.println(deadline);
+            ui.add(deadline, list.size(), onetwo);
+            list.add(deadline);
+            //System.out.println("Now you have " + list.size() + onetwo +  " in the list");
+            //System.out.println("____________________________________________________________");
+            //appendToFile(zenithData, str);
+            refreshData();
+            Answer();
+        }
+        else if (parser.getArr()[0].equals("todo")){
+            try {
+                arr = parser.getArr();
+                Todo todo = new Todo(arr[1]);
+                //System.out.println("____________________________________________________________");
+                //System.out.println("Got it. I've added this task:");//  no following words after
+                //System.out.println(todo);
+                list.add(todo);
+                ui.add(todo, list.size(), onetwo);
+                //System.out.println("Now you have " + list.size() + onetwo +  " in the list");
+                //System.out.println("____________________________________________________________");
+                appendToFile(zenithData, parser.getStr());
+                refreshData();
+                Answer();
+            } catch (ArrayIndexOutOfBoundsException e){
+                ui.specify();
+                Answer();
+            }
+        }
+        else if(parser.getArr()[0].isEmpty()) {
```

Comment:
Same as above. Maybe consider using switch statements to make the code here more concise?